Content.Client/SurveillanceCamera/UI/SurveillanceCameraMonitorWindow.xaml.cs

```diff
@@ -21,8 +22,9 @@ public sealed partial class SurveillanceCameraMonitorWindow : DefaultWindow
 
     [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
     [Dependency] private readonly IResourceCache _resourceCache = default!;
+    [Dependency] private readonly IEntityManager _entityManager = default!;
 
-    public event Action<string>? CameraSelected;
+    public event Action<string, string?>? CameraSelected;
```

Comment:
Personal preference; I like to have events like this be named tuples so you know what each string is. And/or docstring it.